common/tools/eslint-plugin-azure-sdk/tests/rules/ts-config-include.ts

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @file Testing the ts-config-include rule.
+ * @author Wei Jun Tan
+ */
+
+import rule from "../../src/rules/ts-config-include";
+import { RuleTester } from "eslint";
+
+//------------------------------------------------------------------------------
+// Example files
+//------------------------------------------------------------------------------
+
+const exampleTsconfigGood = `{
+  "compilerOptions": {
```

Comment:
nit: do we need the full content of a tsconfig.json for unit test?

Reply:
I included the full `tsconfig.json` to show how this rule would work in a real file and to mimic the style of other relevant unit tests (e.g. `ts-config-exclude.ts`). Would you prefer me using a minified `tsconfig.json` with only the targeted property `include`?

Reply:
I just feel that it's not easy to see what we are testing with the full tsconfig.json but it's fine as-is.